### ISSUE TITLE
renaming default branch from master to main

### ISSUE THREAD

**Issue by yashwanth-l** [CocoaPods/CocoaPods#10263]
* [x ] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

N/A

## What did you expect to happen?

renaming default branch from `master` to `main` to adhere to industry standards

## What happened instead?

Still see `master` is the default branch

## CocoaPods Environment

N/A

## Project that demonstrates the issue

https://github.com/github/renaming

**Comment by dnkoutso** [comment]
Yes we should do that.

**Comment by orta** [comment]
It's still probably worth waiting for GitHub to release the migration tools, so to not break any active PRs

**Comment by yashwanth-l** [comment]
@orta @dnkoutso will that also ensure all the commands like `pod install` or `pod update` will refer the _default branch_ as _main_

**Comment by igor-makarov** [comment]
@yashwanth-l default branch of what? The main spec repo has been called `trunk` for a while now.

**Comment by yashwanth-l** [comment]
> @yashwanth-l default branch of what? The main spec repo has been called `trunk` for a while now.

How would it know _a branch_ is called as `trunk`

**Comment by igor-makarov** [comment]
It's distributed via a CDN, so there's not even a branch that the client accesses. It's an HTTP interface.

**Comment by jamala** [comment]
It looks like private spec repos still use [DEFAULT_SPECS_BRANCH](https://github.com/CocoaPods/Core/blob/9584a94db960979a0182acfbf4de5e677c848847/lib/cocoapods-core/source.rb#L20), which points to master. Would this be changed as part of the migration?

**Comment by billburgess** [comment]
Seeing this issue with private pods as well. I renamed all my `master` branches to `main` and it doesn't see any of the new changes to specs.

**Comment by cameroncooke** [comment]
My organisation has been taken in renaming our branches to `main` and I'm unsure if we can rename our private specs repo without breaking integration. Is there a plan to update this to just use default branch instead of handed `master` as pointed out by @jamala ?

**Comment by nordfogel** [comment]
We encountered an issue by accessing private spec repos via version number after renaming the main branch basically because of the exact same point @jamala kindly mentioned. Any chance that there will be an update tackling this?

**Comment by yashwanth-l** [comment]
Is this still a thing? Should the issue still be open?

**Comment by orta** [comment]
Yeah, the inline default to master is still a feature someone could send a PR for judging by https://github.com/CocoaPods/CocoaPods/issues/10263#issuecomment-758290290. 

WRT switching this repo to use master, now the tooling is pretty good, someone with some time to make sure that doesn't break deploys etc can use the automated tools
